test/api_test.rb

```diff
@@ -6,18 +6,12 @@ def setup
     @url = 'http://my.prestashop.com'
     @url_regex = %r{my[.]prestashop[.]com.*}
     @key = 'VALID_KEY'
-  end
+    @resources = [:customers, :orders, :products]
+    @product_ids = ['1', '2']
 
-  def test_get_resources
     xml = File.open(File.join __dir__, 'xml', 'resources.xml').read
     stub_request(:any, @url_regex).to_return body: xml
 
-    resources = Prestashopper::API.new(@url, @key).resources
-    assert_equal 3, resources.length
-    [:customers, :orders, :products].each {|s| assert_includes resources, s}
-  end
-
-  def test_get_products
```

Comment:
I attempted to keep the existing functionality and complete some additional functionality as closely to the documentation as possible.  New tests have been added for new functionality.